resources/views/checkout/steps/credentials.blade.php

```diff
@@ -5,11 +5,13 @@
     <x-rapidez-ct::progress-bar />
 </div>
 
-<x-rapidez-ct::sections>
-    @include('rapidez-ct::checkout.partials.sections.login')
-    @include('rapidez-ct::checkout.partials.sections.address')
-    @include('rapidez-ct::checkout.partials.sections.shipping')
-</x-rapidez-ct::sections>
+<form id="credentials" v-on:submit.prevent="save(['credentials'], 3)">
```

Comment:
This is the best way to do it? Removing the form from the login and doing a `v-on:click.prevent="go"` there instead of a form submission?

Reply:
I think so. I tried a different workaround in a different project where I kept the forms as they were, and even though it worked it required unnecessary extra code and workarounds. This ended up being the cleanest approach I could come up with.